meshtastic/__main__.py

```diff
@@ -1876,6 +1906,27 @@ def addRemoteActionArgs(parser: argparse.ArgumentParser) -> argparse.ArgumentPar
         action="store_true",
     )
 
+    group.add_argument(
+        "--request-userinfo",
+        help="Request user information from a specific node. "
+        "You need to pass the destination ID as an argument with '--dest'. "
+        "For repeaters, the nodeNum is required.",
+        action="store_true",
+    )
+
+    group.add_argument(
+        "--broadcast-userinfo",
+        help="Broadcast your user information to all nodes in the mesh network.",
+        action="store_true",
+    )
+
+    group.add_argument(
+        "--send-userinfo",
```

Comment:
Let's have this be available as `--send-user` as well, to match the protobuf name more closely.

```suggestion
        "--send-userinfo", "--send-user",
```